pkg/controller/externalippool/validate.go

```diff
@@ -77,6 +82,48 @@ func (c *ExternalIPPoolController) ValidateExternalIPPool(review *admv1.Admissio
 		Result:  result,
 	}
 }
+
+func validateIPRangesAndSubnetInfo(ipRanges []crdv1beta1.IPRange, subnetInfo *crdv1beta1.SubnetInfo) (string, bool) {
+	if subnetInfo == nil {
+		return "", true
+	}
+	gatewayIP := net.ParseIP(subnetInfo.Gateway)
```

Comment:
We need not to check it is a valid v4 or v6 IP?

Reply:
The CRD schema has validated it.